lib/spec/unit/facter_spec.rb

```diff
@@ -131,6 +131,18 @@
       expect(Facter::Util::Resolution.which('foo')).to eq('/usr/bin/foo')
       expect(Facter::Util::Resolution.exec('foo')).to eq('bar')
     end
+
+    it 'Facter._load_external only skips external facts loading when called with false' do
+      Facter.search_external([
+        File.expand_path('../../../lib/tests/fixtures/facts/external/yaml'),
+      ])
+      Facter.load_external(false);
+      facts = Facter.to_hash
+      Facter.load_external(true);
+      expect(facts['yaml_fact1']).to be_nil
+      facts = Facter.to_hash
+      expect(facts['yaml_fact1'] == "foo")
+    end
```

Comment:
Can you split this in two examples? each having its own expect? One to test `Facter.load_external(false)` and the other `Facter.load_external(true)`

```ruby

describe '.load_external' do
  context 'when set to false'
	it '' do
    end
  end
  
  context 'when set to true'
	it '' do
    end
  end
end
```